Components/MetaphorUI.css

```diff
@@ -2,7 +2,19 @@
 @import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700;800&display=swap");
 
 :root {
-  --fontBlack: rgb(24, 24, 24);
+  --fontBlack: #181818;
+  --fontGray: #616060;
+  --colRed: #db2b39;
+  --colRed2: #dd3434;
+  --colViolet: #b91372;
+  --colBlue: #998fc7;
+  --colWhite: #f1f1f1;
+  --colSuccess: #ee8434;
```

Comment:
Shouldn't success have color related to green? At the moment its similar to warning.